Description: id and hash for Telegram API
 I put them here, because it does not make sense to hide them. You always can
 disassemble binary and extract these values. Moreover I should publish, in
 spite of a caution on application page, to guarantee reproducible build.
Author: Nicholas Guriev <guriev-ns@ya.ru>
Forwarded: not-needed
Last-Update: 2017-07-06

--- a/Telegram/SourceFiles/config.h
+++ b/Telegram/SourceFiles/config.h
@@ -252,6 +252,9 @@ w/CVnbwQOw0g5GBwwFV3r0uTTvy44xx8XXxk+Qkn
 
 #ifdef CUSTOM_API_ID
 #include "../../../TelegramPrivate/custom_api_id.h" // Custom API id and API hash
+#elif defined DEBIAN_API_ID
+static const int32 ApiId = 50322; // API id and hash for Debian package
+static const char *ApiHash = "9ff1a639196c0779c86dd661af8522ba";
 #else
 static const int32 ApiId = 17349;
 static const char *ApiHash = "344583e45741c457fe1862106095a5eb";
